other/upgrade.php

```diff
@@ -665,16 +686,16 @@ function load_lang_file()
 		list($_SESSION['upgrader_langfile']) = array_keys($detected_languages);
 
 		// If we have English and some other language, use the other language.
-		if ($_SESSION['upgrader_langfile'] == 'Install.english.php' && count($detected_languages) > 1) {
-			list(, $_SESSION['upgrader_langfile']) = array_keys($detected_languages);
+		if ($_SESSION['upgrader_langfile'] == 'Install.en_US.php' && count($detected_languages) > 1) {
```

Comment:
```suggestion
		if ($_SESSION['upgrader_langfile'] == 'en_US/Install.php' && count($detected_languages) > 1) {
```

Again, I'm not sure if this suggestion will work.

Reply:
I got an idea that should work for both install/upgrade. It is, we only need the language.  No reason to specify install.php anywhere.  I need to make all the changes and test this.

Reply:
@Sesquipedalian I sent up some changes that should still work.  Note that the upgrade.php logic isn't complete.  I have to change over the UTF-8 stuff somehow.  I say somehow because there isn't a one-to-one translation on those.